docs/migration_guide.md

```diff
@@ -91,8 +96,20 @@ For any reason, if you want to roll back to the previous version of Harbor, foll
 2. Restore database from backup file in `/path/to/backup` . 
 
     ```
-    docker run -ti --rm -e DB_USR=root -e DB_PWD=xxxx -v /data/database:/var/lib/mysql -v /path/to/backup:/harbor-migration/backup vmware/harbor-db-migrator:[tag] restore
+    docker run -it --rm -e DB_USR=root -e DB_PWD={db_pwd} -v ${harbor_db_path}:/var/lib/mysql -v ${harbor_cfg_path}:/harbor-migration/harbor-cfg -v ${backup_path}:/harbor-migration/backup vmware/harbor-migrator:[tag] restore
+    ```
+ 
+    **NOTE:** By default, the migrator handles the restore for database and cfg. If you want to restore DB or CFG only, 
```

Comment:
make it consistent in one sentence, it should be both database or DB, cfg or CFG.